packages/mds-geography/api.ts

```diff
@@ -0,0 +1,185 @@
+import express from 'express'
+import db from '@mds-core/mds-db'
+
+import { pathsFor, ServerError, NotFoundError } from '@mds-core/mds-utils'
+import { geographyValidationDetails } from '@mds-core/mds-schema-validators'
+import log from '@mds-core/mds-logger'
+
+import { checkAccess } from '@mds-core/mds-api-server'
+
+function api(app: express.Express): express.Express {
+  app.get(
+    pathsFor('/geographies/meta/'),
+    checkAccess(scopes => scopes.includes('policies:read')),
+    async (req, res) => {
+      const get_read_only = req.query === 'true'
+
+      log.info('read /geographies/meta', req.query)
+      try {
+        const metadata = await db.readBulkGeographyMetadata({ get_read_only })
+        res.status(200).send(metadata)
+      } catch (err) {
+        await log.error('failed to read geography metadata', err)
+        res.status(404).send({
+          result: 'not found'
+        })
+      }
+    }
+  )
+
+  app.get(
+    pathsFor('/geographies/:geography_id'),
+    checkAccess(scopes => scopes.includes('policies:read')),
+    async (req, res) => {
+      log.info('read geo', JSON.stringify(req.params))
+      const { geography_id } = req.params
+      log.info('read geo', geography_id)
+      try {
+        const geography = await db.readSingleGeography(geography_id)
+        res.status(200).send(geography)
+      } catch (err) {
+        await log.error('failed to read geography', err.stack)
```

Comment:
Need better error handling here, just because it throws an error doesn't mean it's a 404, the DB could barf somehow. Should also probably use the `NotFoundError` in `mds-utils/exceptions.ts` to get a response body which is congruent with our other APIs